test/jdk/ProblemList.txt

```diff
@@ -796,7 +796,6 @@ java/awt/print/PrinterJob/ScaledText/ScaledText.java 8231226 macosx-all
 java/awt/font/TextLayout/TestJustification.html 8250791 macosx-all
 java/awt/TrayIcon/DragEventSource/DragEventSource.java 8252242 macosx-all
 java/awt/FileDialog/DefaultFocusOwner/DefaultFocusOwner.java 7187728 macosx-all,linux-all
-java/awt/FileDialog/RegexpFilterTest/RegexpFilterTest.html 7187728 macosx-all,linux-all
```

Comment:
What about [JDK-7187728](https://bugs.openjdk.org/browse/JDK-7187728)? It's still unresolved.

Reply:
It is still open because there is another non-applet test `java/awt/FileDialog/DefaultFocusOwner/DefaultFocusOwner.java` that needs attention.
Since that test sprint is regarding the elimination of applet tests, it is out of scope.
Hence the new bug id for this test.

[JDK-8327835](https://bugs.openjdk.org/browse/JDK-8327835)